src/planning/include/planning/planning.hpp

```diff
@@ -63,21 +63,28 @@ class Planning : public rclcpp::Node {
   int smoothing_spline_precision_;
   bool publishing_visualization_msgs_;
   bool using_simulated_se_ = false;
+  bool recieved_first_track = false;
+  bool recieved_first_pose = false;
+  std::vector<Cone *> cone_array;
   /**< Subscription to vehicle localization */
   rclcpp::Subscription<custom_interfaces::msg::VehicleState>::SharedPtr vl_sub_;
   /**< Subscription to track map */
   rclcpp::Subscription<custom_interfaces::msg::ConeArray>::SharedPtr track_sub_;
   /**< Local path points publisher */
   rclcpp::Publisher<custom_interfaces::msg::PathPointArray>::SharedPtr local_pub_;
   /**< Publisher for the final path*/
-  rclcpp::Publisher<visualization_msgs::msg::MarkerArray>::SharedPtr visualization_pub_;
+  rclcpp::Publisher<visualization_msgs::msg::Marker>::SharedPtr visualization_pub_;
```

Comment:
Why?

Reply:
It's now a Marker with a line, I'd say it's better than the list of markers, it differentiates more to the cones